analyzers.yml

```
@@ -1,5 +1,10 @@
 analyzers:
   # In alphabetical order.
+  actionlint:
+    name: Actionlint
```

[suggestion] According to the [official readme](https://github.com/rhysd/actionlint/blob/f05f9da49f93decc2ac8e61b1f951ed19b7303a7/README.md), `actionlint` seems to be correct instead of `Actionlint`.

```suggestion
    name: actionlint
```

In addition, can you please write a Goodcheck rule to detect such a typo in the following file? ⬇️ 

https://github.com/sider/goodcheck-rules/blob/ceda654aa6097b8d8c9d7e53e36a05a7b71584a1/rules/typo.yml

---

`Actionlint` changed to `actionlint`. For the goodcheck rule, let me open another PR.

---

Goodcheck rule here https://github.com/sider/goodcheck-rules/pull/62

---

I don't have write permission to that repo, can somebody grant it to me, or waive the Sider issue there?